### ISSUE TITLE
`--createstub` outputs one extra colon

### ISSUE THREAD

**Issue by InSyncWithFoo** [microsoft/pyright#8044]
Steps to reproduce (Linux):

```shell
$ pip install pyright a-n-plus-b
$ mkdir typings && chmod 444 typings
$ pyright --createstub a_n_plus_b
```

Output:

```text
Could not create typings subdirectory '/workspaces/test/typings/a_n_plus_b'
Error occurred when creating type stub: : Could not create typings subdirectory '/workspaces/test/typings/a_n_plus_b'
```

There should be only one colon. The offending part is probably [this one](https://github.com/microsoft/pyright/blob/c7fbc82337d8d9aff55a82bb9a8ddbc0b016e8be/packages/pyright-internal/src/pyright.ts#L447-L452):

```ts
try {
    // ...
} catch (err) {
    let errMessage = '';
    if (err instanceof Error) {
        errMessage = ': ' + err.message;
    }

    console.error(`Error occurred when creating type stub: ` + errMessage);
    // ...
}
```

**Comment by erictraut** [comment]
Thanks for the bug report. This will be addressed in the next release.

**Comment by erictraut** [comment]
This is addressed in pyright 1.1.366.
